sunpy/physics/differential_rotation.py

```diff
@@ -144,7 +146,7 @@ def diff_rot(duration: u.s, latitude: u.deg, rot_type='howard', frame_time='side
 
     return Longitude(rotation.to(u.deg))
 
-
+@deprecated(since="5.1.1", message="We will be moving this", alternative="sunpy.coordinates.utils._validate_observer_args")
```

Comment:
1) You only need to do this for public API, the leading underscore marks this as private API (its a style choice in Python).

2) This will come out in sunpy 6.0 and not 5.1.1.

Reply:
Okay I will fix this. I thought since reflects the version in which the deprecation happened.